src/main/perl/FileEditor.pm

```diff
@@ -47,24 +47,79 @@ the class constructor.
 =item new
 
 Returns a new object it accepts the same arguments as the constructor
-for C<CAF::FileWriter>
+for C<CAF::FileWriter> with one additional option: 
+
+=over
+
+=item source 
+
+This option, when present, must be a file name whose contents will be used
```

Comment:
with current code, the source for Editor can also be a pipe (and maybe mention that if it is a pipe, it is always considered more recent)